_posts/2019-11-14-april-2020-workshop.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: The First US-RSE Association Workshop
+posted_by: Ian Cosden
+tags: [workshop]
+---
+
+We are excited to announce that through a generous grant from the Alfred P. Sloan Foundation, we will be able to hold the first US-RSE Association community building workshop, April 21-22, 2020 in Princeton, NJ.
+This won't be an attempt to be an RSE conference, but a workshop focused on planning the best path forward to grow the US-RSE Association.
+
+
+The workshop will have breakout sessions focused on a number of different topics pertinent to the US-RSE community.
+We’ll be working together to generate material/content for the us-rse.org website that promotes and supports current/future RSEs in alignment with the [mission](https://us-rse.org/mission/) of the US-RSE Association, and to plan follow-up activities to grow the community.  
+More specific workshop information and details will follow in the coming weeks.
+
+You can find out more and express interest in attending by filling out the form here: https://forms.gle/tzronZUGdUTT9zzb9.
+Even though we have to keep this workshop small, remember that there are many ways to get more involved in the community and help grow the US-RSE Association.
```

Comment:
This sounds like a foreshadow of "Sorry we can't afford to invite you!" I wouldn't state it directly here (it might discourage people from expressing interest if they don't feel worthy) but just delete this sentence, and then start with a new section - "Other Ways to Get Involved" (and then proceed with talking about the website and social media). If that's a header section, it will also work well to grab the attention of folks who maybe don't want to go to the conference, but would want to get involved in other ways.

Reply:
And after reading I'm seeing the section at the bottom about wanting diversity, having limited number of seats, that's the perfect spot to put this information (after a person has expressed interest!)

Reply:
Ok, this is the only thing I haven't addressed.  I'm not exactly sure what you are suggesting with your second comment @vsoch.  It's the perfect spot to put what information?

Reply:
Sorry different section of the file, at the way bottom you say:

> We will strive to accommodate as many as possible, but workshop space is limited and we are committed to ensuring a broadly diverse participant group, both in terms of traditional diversity measures, but also in terms of research domain and institutional representation. We feel strongly that a diverse community represents more potential members, has more sources for ideas, and has fewer shared assumptions that might hinder the growth of the US-RSE Association. 

And I think that's the right spot to mention the limited attendance thing (as you already do) but not before they have expressed interest!

Reply:
TLDR: no further action, what you have, and where you placed it, is spot on. I'd just nix the earlier mention, per my first comment.

Reply:
Actually I rearranged it.